packages/core/data-transfer/.npmignore

```diff
@@ -0,0 +1,133 @@
+#############################
+# DATA TRANSFER
+############################
+
+*.js.map
+src/
+types/
```

Comment:
I think we want to include at least 'types' so that the types are available, and the other two if we want to allow debugging.